src/test/components/file-contents.tsx

```diff
@@ -0,0 +1,96 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Is it ok to name these files `foo.test.tsx` or something like that? I think it's nice to see at a glance when I'm working in the source and test files